src/cls/rbd/cls_rbd.h

```diff
@@ -164,4 +164,15 @@ struct cls_rbd_snap {
 };
 WRITE_CLASS_ENCODER(cls_rbd_snap)
 
+namespace group {
```

Comment:
@dillaman I need this function to be available in unit tests: ceph/src/test/cls_rbd/test_cls_rbd.cc:2308:21. Since the test was not about building snap_key I didn't want to duplicate this code. What is your opinion?

Reply:
OK -- leave it then.